manifests/m/Microsoft/VC++2013Redist-x64/12.0.40649.5/Microsoft.VC++2013Redist-x64.installer.yaml

```
@@ -19,6 +19,8 @@ Installers:
   Scope: machine
   InstallerUrl: https://download.microsoft.com/download/c/c/2/cc2df5f8-4454-44b4-802d-5ea68d086676/vcredist_x64.exe
   InstallerSha256: 8588EB697EB2049344E6206D2B66FF63104F1C55E553621AB8ECC504D6B9E9D4
+  InstallerSwitches:
+    Custom: /norestart
   UpgradeBehavior: install
```

```suggestion
  UpgradeBehavior: uninstallPrevious
```

https://github.com/microsoft/winget-cli/issues/1424#issuecomment-910312191